### ISSUE TITLE
On Paper 1.20.4, java.lang.IncompatibleClassChangeError: Found class org.bukkit.inventory.InventoryView, but interface was expected

### ISSUE THREAD

**Issue by Poikilos** [mehboss/CustomRecipes#57]
### What happened?

java.lang.IncompatibleClassChangeError: Found class org.bukkit.inventory.InventoryView, but interface was expected

...but the plugin works (and #55 and #56 fixed)

### What was supposed to happen?

Perform check to allow cross-version compatibility

### How do I reproduce this issue?

```bash
1. Install plugin
2. Load Paper 1.20.4 server
3. Join server
```

### Relevant log output

https://pastebin.com/0uVDtPfN

### Server Version

Paper 1.20.4

### Plugin Version

1.9.2

### Server Mode

Online Mode

### Issue Confirmation

- [X] I have downloaded and tried the latest version of custom recipes and the issue still persists.
- [X] I have provided sufficient information for the issue to be replicated.

**Comment by mehboss** [comment]
`[04:19:40 INFO]: [Multiverse-Core] Version 4.3.1-b861 (API v24) Enabled - By dumptruckman, Rigby, fernferret, lithium3141 and main--
[04:19:40 INFO]: [ViaVersion] Enabling ViaVersion v5.0.3
[04:19:40 INFO]: [ViaVersion] ViaVersion detected server version: 1.20.3-1.20.4 (765)
[04:19:40 INFO]: [PlaceholderAPI] Enabling PlaceholderAPI v2.11.3
[04:19:40 INFO]: [PlaceholderAPI] Fetching available expansion information...
[04:19:40 INFO]: [TPRituals] Enabling TPRituals v1.5
[04:19:40 INFO]: [ViaBackwards] Enabling ViaBackwards v5.0.3
[04:19:40 INFO]: [CustomRecipes] Enabling CustomRecipes v1.9.2
[04:19:41 INFO]: [PlaceholderAPI] Successfully registered expansion: customrecipes [1.9.2]
[04:19:41 INFO]: [CustomRecipes] Made by MehBoss on Spigot. For support please PM me and I will get back to you as soon as possible!
[04:19:41 INFO]: [CustomRecipes] Loading Recipes..
[04:19:41 WARN]: [CustomRecipes] Attempting to add the recipe EnchantedDiamond..
[04:19:41 WARN]: [CustomRecipes] Applying displayname..
[04:19:41 WARN]: [CustomRecipes] Displayname: &aEnchanted Diamond
[04:19:41 WARN]: [CustomRecipes] Ingredient Amount: 3
[04:19:41 WARN]: [CustomRecipes] Ingredient Displayname: false
[04:19:41 WARN]: [CustomRecipes] Ingredient Identifier: none
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: false
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: none
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: false
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: none
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: none
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: none
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: false
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: none
[04:19:41 WARN]: [CustomRecipes] HandlingIngredient: false
[04:19:41 WARN]: [CustomRecipes] Successfully added EnchantedDiamond with the amount output of 1
[04:19:41 INFO]: [CustomRecipes] Loaded 1 recipes.
[04:19:41 INFO]: [PlaceholderAPI] Placeholder expansion registration initializing...
[04:19:41 INFO]: Running delayed init tasks
[04:19:41 INFO]: Done (4.701s)! For help, type "help"
[04:19:41 INFO]: [ViaVersion] Finished mapping loading, shutting down loader executor!
[04:19:41 INFO]: Timings Reset
[04:19:41 INFO]: [PlaceholderAPI] Successfully registered expansion: player [2.0.5]
[04:19:41 INFO]: [PlaceholderAPI] Successfully registered expansion: localtime [1.2]
[04:19:41 INFO]: 2 placeholder hook(s) registered! 1 placeholder hook(s) have an update available.
[04:19:41 INFO]: [TPRituals] Checking for updates..
[04:19:41 INFO]: [TPRituals] We are all up to date with the latest version. Thank you for using teleportation rituals :)
[04:19:41 INFO]: [CustomRecipes] Checking for updates..
[04:19:41 INFO]: [CustomRecipes] We are all up to date with the latest version. Thank you for using custom recipes :)
[04:19:42 INFO]: [PlaceholderAPI] An update for PlaceholderAPI (v2.11.6) is available at:
[04:19:42 INFO]: [PlaceholderAPI] https://www.spigotmc.org/resources/placeholderapi.6245/
version
[04:22:22 INFO]: Checking version, please wait...
[04:22:22 INFO]: This server is running Paper version git-Paper-336 (MC: 1.20.4) (Implementing API version 1.20.4-R0.1-SNAPSHOT) (Git: 086ca61)
You are 161 version(s) behind`
This is my log startup log on paper 1.20.4 according to your reproduction steps.

**Comment by Poikilos** [comment]
Ok, I cannot reproduce the error. I'll let you know if I run into it again. Thanks.

**Comment by Poikilos** [comment]
For future reference I made a linux program to run various server versions (may make Windows version) using GUI buttons: https://github.com/Poikilos/MoreServers

**Comment by mehboss** [comment]
I was able to reproduce by trying the recipe book. Seems like it broke in 1.20. I have a fix, just waiting to fix the other issue. 

Thank you

**Comment by mehboss** [comment]
It seems that version 1.20.4 is the only affected version. InventoryView changed from a class to an interface and no routing was present in that version. I am unable to come up with a fix for that particular version.

**Comment by mehboss** [comment]
Issue has been resolved in 1.9.3 [release](https://github.com/mehboss/CustomRecipes/commit/2064fb2b47d4ad0694a240ba177c26cd9dc6e6de). Will be uploaded to spigot soon.

Thank you
